tests/conftest.py

```diff
@@ -24,7 +24,13 @@ def _dataset_names() -> list[str]:
 @pytest.fixture(scope="session")
 def project_path() -> Generator[Path, None, None]:
     """Return a temporary directory path that is used as the project directory for the entire test."""
-    with TemporaryDirectory() as project_path:
+    # Get the root directory of the project
+    # We want to create the temporary directory in the root directory of the project.
+    # This is useful in terms of accessing the test files in the project directory.
+    # This tmp directory will be deleted after all the tests are run.
+    tmp_dir = Path(__file__).parent.parent
```

Comment:
maybe we could add a common folder to the path to avoid cluttering the root directory in case the tempdir is not properly removed for some reason.
```suggestion
    tmp_dir = Path(__file__).parent.parent / ".tmp"
```